roles/os_update_latest/tasks/main.yml

```diff
@@ -1,3 +1,10 @@
 ---
+- name: Make room for new kernel on /boot
+  command: /usr/bin/package-cleanup --oldkernels --count=2 -y
+
 - name: Update all packages
-  package: name=* state=latest
+  yum:
+    name:  "{{ oul_package_names }}"
+    exclude: "{{ oul_exclude }}"
+    state: latest
+  when: os_update is defined
```

Comment:
Why the when condition? Just don't call this role if you don't intend to update packages?

Reply:
Wrong condition - I'm still an ansible noob.  :D I'll change that to a bool and an appropriate default.